pf/tfbridge/provider_diff.go

```diff
@@ -46,6 +46,11 @@ func (p *provider) DiffWithContext(
 ) (plugin.DiffResult, error) {
 	ctx = p.initLogging(ctx, p.logSink, urn)
 
+	checkedInputs, err := applyIgnoreChanges(priorStateMap, checkedInputs, ignoreChanges)
```

Comment:
This is very different than doIgnoreChanges in the pre-PF bridge that looks at news and olds post Diff, post planning the resource change (which also applies defaults and such erm).. I think we can construct test cases where this will behave a bit differently from the current bridged provider behavior, and I think the pressure will be to match existing behavior as we're trying to make as little disruptive changes as possible 😓

Reply:
Could not construct a test case here that would be relevant. One possibility if state upgrade is migrating the format (or type) of a property P that's being ignored. In that case copying from state to news will bypass the migration on the property and news will have a bad format, which will bork TF processing.  Sounds sufficiently niche to wait on that until such time as we need to deal with it.